src/components/dapps/ModalCreateDapps.vue

```diff
@@ -388,12 +239,15 @@ export default defineComponent({
 
     const openOption = ref(false);
 
-    const unit_d = 3;
-    const decimal = 12;
+    const unitToken = inject('unitToken', '');
+    const decimal = inject('decimal', 12);
+
+    const selectUnitEndowment = ref(unitToken);
+    const selectUnitGas = ref('micro');
 
     const formData = reactive<FormData>({
-      endowment: bnToBn(plasmUtils.reduceDenomToBalance(27, unit_d, decimal)),
-      weight: new BN(200000000000),
+      endowment: new BN(27000),
```

Comment:
it's good to use `BN` for all balance types, and all `BN` values should represent the minimum possible token unit (`femto`). The value 27,000 is too big to be the default denominator (ex: `PLM`) and too small to be in `femto`. Please ensure that all balance values are stored as `BN` of `femto` and only convert the numbers either from or to `femto` (like how Ethereum balances are always read as `wei` instead of `ETH).

Reply:
Right, but I confirmed that when a transaction is sent, it is all converted to `femto`. The reason why I set 27 killo unit is that related with https://github.com/PlasmNetwork/plasm-apps-vue/pull/43#discussion_r664273293 . to upload my `flipper.contract` , it requires above 25.6 Kilo unit. anyway we can set zero endowment as default.